transform/models/marts/fact_stars.sql

```diff
@@ -0,0 +1,6 @@
+SELECT 
+    DATE_TRUNC('day', event_date) AS date,
```

Comment:
Same: Let's keep the date as is here for now (although makes perfect sense to truncate).